_data/api-language.yml

```diff
@@ -17,6 +16,9 @@ microsoft:
   fil: tl # Macrolanguage, Microsoft for Tagalog
   mww: hmn # Macrolanguage, Microsoft for Hmong
   cz: cs # Czech, Omniscien
+  me: cnr # Montenegrin, Niutrans
+  mo: mn # Mongolian, Niutrans
+  jy: ka # Georgian, Niutrans
```

Comment:
Arguably, these should go under `niutrans:`

It probably doesn't hurt to have them here, but it's hard to imagine another provider doing this exact same terrible deviation from the standard.

Reply:
I'll merge. We can reorganize it later if we decide to go with that option.